spec/workspace-spec.coffee

```diff
@@ -426,6 +426,37 @@ describe "Workspace", ->
       workspace.decreaseFontSize()
       expect(atom.config.get('editor.fontSize')).toBe 1
 
+  describe "::resetFontSize()", ->
+    it "resets the font size to the window's starting font size", ->
+      originalFontSize = atom.config.get('editor.fontSize')
+
+      atom.config.set('editor.fontSize', originalFontSize)
```

Comment:
I don't think this line is needed since the value is read on the previous line and the value isn't changed to `atom.config.set` will ignore it I think since the value hasn't changed.